### ISSUE TITLE
Full GLUE dataset using PyTorch BERT

### ISSUE THREAD

**Issue by leuchine** [NVIDIA/DeepLearningExamples#309]
May I ask whether the PyTorch BERT can be used to fine-tune the full GLUE dataset or not? I only saw the script for MPRC at the moment. Thanks!

**Comment by sharathts** [comment]
GLUE isn't officially supported in PyTorch BERT yet. Training scripts does exist, but hasn't been tested with the latest containers. If interested, you can adapt `run_glue.py` as in `run_squad.py` . Launching script exists in `scripts/run_glue.sh` . But you will have to do a hyperparameter search for the best accuracy.

**Comment by leuchine** [comment]
Thanks for your reply! May I ask is it possible to adapt the hyperparameters from the Google BERT here for other tasks? Does any hyperparameter particularly need to be tuned? Thanks!
